openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateSpaceType.cpp

```diff
@@ -155,6 +159,13 @@ boost::optional<IdfObject> ForwardTranslator::translateSpaceType( SpaceType & mo
     translateAndMapModelObject(equipment);
   }
 
+  // translate IT electric equipment
+  ElectricEquipmentITEAirCooledVector electricEquipmentITEAirCooled = modelObject.electricEquipmentITEAirCooled();
```

Comment:
There won't be any assigned to the space type right?

Reply:
ITE object **can** be assigned to space type in OS, but will be separately assigned to each space when being translated to E+.